sdk/core/azure_core/README.md

```diff
@@ -201,6 +201,20 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 }
 ```
 
+## Performance Testing
+
+Azure Core includes performance benchmarks to help monitor the performance of critical components. These benchmarks are written using the [criterion](https://crates.io/crates/criterion) crate, a statistics-driven benchmarking library.
+
+To run the performance tests:
+
+```bash
+cd sdk/core/azure_core
+cargo bench
+```
+
+The benchmarks measure critical operations like serialization/deserialization, HTTP request processing, and token acquisition to ensure optimal performance of the SDK.
```

Comment:
I'm not seeing the tests for serialization/deserialization, HTTP request processing and token acquisition. Are they coming in a later PR?

Reply:
no this is just to start , i added a simple test to be able to write the automation part ( invoke tests/parse results)

Reply:
Cool. That's what I suspected.